src/index.html

```diff
@@ -10,7 +10,32 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class ="search-form" data-qa="big">
```

Comment:
You have a lot of misformatted attributes with spaces between them -> `class = ` this is not correct, it should be `class=` correct all of such attributes